BetterReads/BetterReads/View Controllers/BookDetailViewController.swift

```diff
@@ -81,12 +98,43 @@ class BookDetailViewController: UIViewController {
         return lb
     }()
     
+    let bottomStackView: UIStackView = {
+        let sv = UIStackView()
+        sv.axis = .vertical
+        sv.distribution = .equalSpacing
+        sv.spacing = 4
+        sv.translatesAutoresizingMaskIntoConstraints = false
+        return sv
+    }()
+    
+    let publisherLabel: UILabel = {
+        let pl = UILabel()
+        pl.text = "Publisher: Little, Brown Books for Young Readers, 2008"
+        pl.font = UIFont(name: "SourceSansPro-Light", size: 16)
+        pl.numberOfLines = 0
+        return pl
+    }()
+    
+    let isbnLabel: UILabel = {
+        let il = UILabel()
+        il.text = "ISBN: 9780316032834"
+        il.font = UIFont(name: "SourceSansPro-Light", size: 16)
+        return il
+    }()
+    
+    let lengthLabel: UILabel = {
+        let ll = UILabel()
+        ll.text = "Length: 768 pages"
+        ll.font = UIFont(name: "SourceSansPro-Light", size: 16)
+        return ll
+    }()
+    
     /// This is a label because it's height can be easily based on it's content (text) inside so scroll view changes accordingly
```

Comment:
Thanks for leaving this comment here.

Reply:
I don't remember writing that comment. I've been set up